src/devices/mindy.ts

```diff
@@ -0,0 +1,366 @@
+            await endpoint.read("genBasic", [0x0240]);
+            await endpoint.read("genBasic", [0x0241]);
+            await endpoint.read("genBasic", [0x0242]);
+
+            await endpoint.read("msCO2", [0x0220]);
+            await endpoint.read("msCO2", [0x0221]);
+            await endpoint.read("msCO2", [0x0222]);
+            await endpoint.read("msCO2", [0x0223]);
+
+            await endpoint.read("msCO2", [0x0230]);
+            await endpoint.read("msCO2", [0x0231]);
+            await endpoint.read("msCO2", [0x0232]);
+
+            await endpoint.read("msIlluminanceMeasurement", [0x0220]);
+            await endpoint.read("msIlluminanceMeasurement", [0x0221]);
+            await endpoint.read("msIlluminanceMeasurement", [0x0222]);
+            await endpoint.read("msIlluminanceMeasurement", [0x0223]);
+            await endpoint.read("msIlluminanceMeasurement", [0x0210]);
+
+            await endpoint.read("msTemperatureMeasurement", [0x0210]);
+            await endpoint.read("msTemperatureMeasurement", [0x0211]);
+
+            await endpoint.read("msRelativeHumidity", [0x0210]);
+
+            await endpoint.read("msPressureMeasurement", [0x0210]);
+        },
+        extend: [
+            m.temperature(),
+            m.humidity(),
+            m.pressure(),
+            m.co2(),
+            m.illuminance(),
+            m.light({configureReporting: true, powerOnBehavior: false, effect: false}),
+            m.identify(),
+            m.numeric({
+                name: "read_interval",
+                unit: "seconds",
+                valueMin: 5,
+                valueMax: 600,
+                access: "ALL",
+                cluster: "genBasic",
+                attribute: {ID: 0x0200, type: Zcl.DataType.UINT16},
+                description: "Read interval of sensors. Default 30",
+                entityCategory: "config",
+            }),
+            m.binary({
+                name: "night_mode",
+                valueOn: ["ON", 1],
+                valueOff: ["OFF", 0],
+                cluster: "genBasic",
+                attribute: {ID: 0x0240, type: Zcl.DataType.BOOLEAN},
+                description: "Turn OFF LED at night",
+            }),
+            m.numeric({
+                name: "night_on_time",
+                unit: "Hr",
+                valueMin: 0,
+                valueMax: 23,
+                cluster: "genBasic",
+                attribute: {ID: 0x0241, type: Zcl.DataType.UINT8},
+                description: "Night mode activation time",
+            }),
+            m.numeric({
+                name: "night_off_time",
+                unit: "Hr",
+                valueMin: 0,
+                valueMax: 23,
+                cluster: "genBasic",
+                attribute: {ID: 0x0242, type: Zcl.DataType.UINT8},
+                description: "Night mode deactivation time",
+            }),
+            m.binary({
+                name: "CO2_control",
```

Comment:
```suggestion
                name: "co2_controll",
```

Use `snake_case` here (also for others)

Reply:
I think I've got it all fixed.